### PR TITLE
Add basic HTML

### PR DESCRIPTION
**Pull request by darlaevans2000**
# Description
Added all basic html needed to start on css in iteration 0

## Type of change
HTML feature
- [ X] New feature (non-breaking change which adds functionality)
# How Has This Been Tested?
Open index.html

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" href="./styles.css">
+  <link href="https://fonts.googleapis.com/css?family=Montserrat:300,400&display=swap" rel="stylesheet">
```
Comment (clairefields15) [inline index.html:7]: Let's add a <title> here!

File: index.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+</head>
+
+<body>
+  <header>
+    <h1> INTENTION TIMER </h1>
+  </header>
+  <main>
+    <section class="left-section">
+      <h2 class="new-activity" New Activity </h2>
+        <section class="left-section">
```
Comment (clairefields15) [inline index.html:17]: Why are there two sections with the class left-section?

File: index.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+          <p class="select-category">Select a category:</p>
+          <button class="study-button" id="studyButton" type="button">Study</button>
+          <button class="meditate-button" id="meditateButton" type="button">Meditate</button>
+          <button class="exercise-button" id="exerciseButton" type="button">Exercise</button>
```
Comment (clairefields15) [inline index.html:21]: Do we need to add a group to all the buttons like with radio buttons so you can't click more than one?

**Review by clairefields15 — CHANGES_REQUESTED** [review]
This looks great! A few comments for changes, and I think the indentation needs help.

File: index.html (hunk 2 above)
Comment (lschnell8) [reply index.html:17]: Other than the double class name here, lookin' good!

**Review by lschnell8 — APPROVED** [review]

**Pull request closed without merging**